templates/terraform/unordered_list_customize_diff.erb

```diff
@@ -0,0 +1,44 @@
+keys := diff.GetChangedKeysPrefix(<%= go_literal(Google::StringUtils.underscore(prop.name)) -%>)
+if len(keys) < 1 {
+	return nil
+}
+
+oldCount, newCount := diff.GetChange("<%= Google::StringUtils.underscore(prop.name) -%>.#")
+var count int
+if oldCount.(int) < newCount.(int) {
```

Comment:
Does it make sense to return here if the counts aren't the same?

Reply:
Duplicates may exist.  Added a comment.